src/components/Layout/RightSidebar.tsx

```diff
@@ -321,6 +321,34 @@ const RightSidebar = () => {
                 </Tooltip>
               </a>
             ))}
+            <a
+              href={`${location.pathname.replace(/\/$/, '')}/index.md`}
+              className="flex h-5 ui-theme-dark group/markdown-link cursor-pointer"
+              onClick={(e) => {
```

Comment:
Consider opening this in a new tab like the other links. It's not external, but it is a disruptive loss of context to replace the window you're on